src/config/window_definition.rs

```diff
@@ -117,11 +137,38 @@ impl StrutDefinition {
     }
 }
 
+// Surface definition if the backend for Wayland is enable
+#[cfg(feature = "wayland")]
+#[derive(Debug, Clone, Copy, Eq, PartialEq, Default)]
+pub struct StrutDefinition {
+    pub layer: WindowStacking,
```

Comment:
This is rather confusing, as the stacking is otherwise defined on the EwwWindowDefinition,.... it should probably stay there.